aptos-move/block-executor/src/scheduler_v2.rs

```diff
@@ -180,6 +192,11 @@ impl SchedulerProxy {
     }
 }
 
+// Const flag values that may be stored as committed marker for a transaction.
+const NOT_COMMITTED: u8 = 0;
+const PENDING_COMMIT_HOOK: u8 = 1;
```

Comment:
I am not sure I see a reason to distinguish between PENDING_COMMIT_HOOK and COMMITTED.

Reply:
So, committed_marker basically helps with arming the lock, but difference between PENDING and COMMITTED is just to check caller uses correctly.